tests/integ/ml/test_transformer.py

```diff
@@ -275,19 +275,36 @@ def test_sparse_merge_onehotencoder(session):
     )
 
 
-def test_input_cols_output_cols_not_match(session):
+def test_input_cols_output_cols_negative(session):
     df = session.create_dataframe(
         [[1, 4, 5], [2, 4, 5], [2, 3, 6]], schema=["a", "b", "c"]
     )
     onehotencoder = OneHotEncoder()
+    # test no input column
+    with pytest.raises(ValueError) as ex_info:
+        onehotencoder.transform(df)
+    assert "Input column can not be empty" == str(ex_info.value)
```

Comment:
```suggestion
    assert "Input column can not be empty" in str(ex_info.value)
```
The same for other assertion of strings